readme.md

```diff
@@ -2,7 +2,7 @@
 An open source re-implementation of Roller Coaster Tycoon 2. A construction and management simulation video game that simulates amusement park management.
 
 ### Build Status
-|             | Windows | Linux / OSX | Download |
+|             | Windows | Linux / Mac | Download |
```

Comment:
I would say change the 3 instances of Mac OS X as headers to say just Mac, and the other instance of OS X to say macOS instead.